storage/ndb/rest-server/rest-api-server/internal/integrationtests/feature_store/handler_test.go

```diff
@@ -0,0 +1,690 @@
+/*
```

Comment:
same as before do use the constants defined in the testdb package for database names. you can also add more constants there for table names.